### DISCUSSION TITLE
Does a list of supported models for each kind of device exists?

### DISCUSSION THREAD

**Discussion by agarcialeon** [koreader/koreader#9106] [Q&A] [answered]
I've been searching through the MobileRead forums trying to find if my device has any koreader release that currently works with no luck.

My device is the Pocketbook 614W (Basic 2), but it would be really great if the github repo or the koreader site could have an updated list of the models of each kind of device and its compatibility status.

**Comment by poire-z** [comment]
We'll probably be too lazy to maintain a Wiki page :)

But one can see what is currently supported in the device specific bits of code:
https://github.com/koreader/koreader/tree/master/frontend/device
https://github.com/koreader/koreader/blob/master/frontend/device/pocketbook/device.lua
and your device seems to be supported (or at least "known", meaning KOReader will start - some features may have issues):
https://github.com/koreader/koreader/blob/118e063ae29eb86b35ffb331ac368137f67102c1/frontend/device/pocketbook/device.lua#L653-L654
https://github.com/koreader/koreader/blob/118e063ae29eb86b35ffb331ac368137f67102c1/frontend/device/pocketbook/device.lua#L420-L428

**Comment by Frenzie** [comment]
I'm not sure if a page prone to become outdated unless someone's right on top of it is a good idea unless we can automate it somehow. Basically if you search in frontend/device/pocketbook/device.lua you should be able to find your model (or not).

https://github.com/koreader/koreader/blob/118e063ae29eb86b35ffb331ac368137f67102c1/frontend/device/pocketbook/device.lua#L420-L428

**Comment by agarcialeon** [comment, marked as answer]
I agree in that such a page could become quickly outdated, but trying to have a simple page with models that at shows at least which ones have a minimal working version could be enough to help novel users of the application the will to contribute with testing or other tasks instead of searching through the code for references to its device. 

Anyway, if you find it to be extra work, maybe referencing those device.lua files in the readme or website could be a simple solution to help users find if the application offers support for its device.

**Reply by Frenzie** [reply]
I added a note here: https://github.com/koreader/koreader/wiki/Installation-on-PocketBook-devices

I'll leave it to PB people to update that if desired.

Is the situation on PocketBook different than on, say, Kobo and Kindle? Because basically everything is supported unless it's brand new. I thought that was basically the same and as such there's no need to search for anything.

There do seem to be an awful lot of extremely minor variations.

**Reply by NiLuJe** [reply]
Yeah, it should be basically everything, and if something's missing, we'd rather hear about it via a bug report ;).

**Reply by agarcialeon** [reply]
I can't speak about kindle or kobo devices because I only own a pocketbook but while searching on Mobile Read forums I couldn't find a clear response mostly because mine has touch controls, no backlight and can't update to the latest PB's software (5.X series). 

So I though asking for a table or list of already tested models could be a good idea instead of trying it on my device directly and maybe suffer a crash or something similar without the prior knowledge to solve those situations.

**Reply by agarcialeon** [reply]
Anyway thanks for the addition to the wiki. It's better than nothing.

**Reply by Frenzie** [reply]
Actually given @NiLuJe's response I think it would be more appropriate to remove that again since it creates the incorrect impression that there's a list *to* check.

**Reply by NiLuJe** [reply]
Yeah, the usual approach is "try it and see what happens", and to let us know if anything goes wrong ;).
